composer.json

```diff
@@ -15,14 +15,14 @@
             "email": "renato.marinho@s2move.com"
         }
     ],
-    "require": {
+  "require": {
         "php": "^8.0",
-        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0|^10.0"
+        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5 || ^9.0",
-        "orchestra/testbench": "^4.0 || ^5.0 || ^6.0 || ^7.0",
-        "squizlabs/php_codesniffer": "^3.6",
```

Comment:
The addition of `^3.7` isn't needed. Everything matching `^3.7` will be matched by `^3.6`

For a nice view see the difference between https://semver.madewithlove.com/?package=squizlabs%2Fphp_codesniffer&constraint=^3.6&stability=stable and https://semver.madewithlove.com/?package=squizlabs%2Fphp_codesniffer&constraint=^3.6&stability=stable